sdk/requirements.txt

```diff
@@ -12,4 +12,4 @@ protobuf>=3.0.0,<4.0.0dev
 python_dateutil>=2.5.3
 requests
 urllib3>=1.25.3
-caraml-auth-google @ git+https://github.com/caraml-dev/caraml-sdk.git@a662528#egg=caraml-auth-google&subdirectory=packages/caraml-auth-google
\ No newline at end of file
+caraml-auth-google==0.0.0.post3
```

Comment:
Just realised we have `google-auth>=2.16.2` dependency here. Isn't it handled by `caraml-auth-google` ? Can we remove the Google auth dependency from this file?

Reply:
Good point, removing it then! Thanks!